From c94720f43f28376562deb4300c8dcd266851f85a Mon Sep 17 00:00:00 2001
From: Romain Naour <romain.naour@gmail.com>
Date: Wed, 21 Oct 2020 21:57:39 +0200
Subject: [PATCH] src/local.mk: add $(LIBHURD)

---
 src/local.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/local.mk b/src/local.mk
index 3894556f3..abbe3163e 100644
--- a/src/local.mk
+++ b/src/local.mk
@@ -91,7 +91,7 @@ remove_ldadd =
 # must precede $(LIBINTL) in order to ensure we use GNU getopt.
 # But libcoreutils.a must also follow $(LIBINTL), since libintl uses
 # replacement functions defined in libcoreutils.a.
-LDADD = src/libver.a lib/libcoreutils.a $(LIBINTL) lib/libcoreutils.a
+LDADD = src/libver.a lib/libcoreutils.a $(LIBINTL) lib/libcoreutils.a $(LIBHURD)
 
 # First, list all programs, to make listing per-program libraries easier.
 # See [ below.
-- 
2.25.4

